encoders/image/PaddleHubEncoder/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jina
+paddlehub>=1.8.2
+paddlepaddle
```

Comment:
please run `docker build -t 'jinaai/hub.encoder.image.imagepaddlehub'  locally and ensure the docker file is built successfully.

Reply:
Yes, on running this as well as ```docker build -f Dockerfile -t jinaai/hub.executors.b .``` I'm facing an issue:
In post_init of __init__: 'context' is not recognised in the module and also a Network Error.

Log here:

```
=================================== FAILURES ===================================
______________________ test_imagepaddlehubencoder_encode _______________________

mocker = <MagicMock name='Module' id='139683180982928'>

    @mock.patch('paddlehub.Module', return_value=MockModule())
    def test_imagepaddlehubencoder_encode(mocker):
>       _test_imagepaddlehubencoder_encode()

tests/test_paddlehub.py:51: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_paddlehub.py:42: in _test_imagepaddlehubencoder_encode
    encoder = get_encoder()
tests/test_paddlehub.py:31: in get_encoder
    return PaddleHubEncoder(metas=metas)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:58: in __call__
    getattr(obj, '_post_init_wrapper', lambda *x: None)(m, r)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:161: in _post_init_wrapper
    self.post_init()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <workspace.PaddleHubEncoder object at 0x7f0a59ae52d0>

    def post_init(self):
        super().post_init()
        import paddlehub as hub
        module = hub.Module(name=self.model_name)
>       inputs, outputs, self.model = module.context(trainable=False)
E       AttributeError: 'MockModule' object has no attribute 'context'

__init__.py:70: AttributeError
----------------------------- Captured stdout call -----------------------------
PaddleHubEncode@12[I]:post initiating, this may take some time...
PaddleHubEncode@12[I]:post initiating, this may take some time takes 0.001 secs
___________________ test_imagepaddlehubencoder_save_and_load ___________________

mocker = <MagicMock name='Module' id='139682429590352'>

    @mock.patch('paddlehub.Module', return_value=MockModule())
    def test_imagepaddlehubencoder_save_and_load(mocker):
>       encoder = get_encoder()

tests/test_paddlehub.py:56: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_paddlehub.py:31: in get_encoder
    return PaddleHubEncoder(metas=metas)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:58: in __call__
    getattr(obj, '_post_init_wrapper', lambda *x: None)(m, r)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:161: in _post_init_wrapper
    self.post_init()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <workspace.PaddleHubEncoder object at 0x7f0a5998db10>

    def post_init(self):
        super().post_init()
        import paddlehub as hub
        module = hub.Module(name=self.model_name)
>       inputs, outputs, self.model = module.context(trainable=False)
E       AttributeError: 'MockModule' object has no attribute 'context'

__init__.py:70: AttributeError
----------------------------- Captured stdout call -----------------------------
PaddleHubEncode@12[I]:post initiating, this may take some time...
PaddleHubEncode@12[I]:post initiating, this may take some time takes 0.001 secs
_______________ test_imagepaddlehubencoder_save_and_load_config ________________

mocker = <MagicMock name='Module' id='139682429631376'>

    @mock.patch('paddlehub.Module', return_value=MockModule())
    def test_imagepaddlehubencoder_save_and_load_config(mocker):
>       encoder = get_encoder()

tests/test_paddlehub.py:72: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_paddlehub.py:31: in get_encoder
    return PaddleHubEncoder(metas=metas)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:58: in __call__
    getattr(obj, '_post_init_wrapper', lambda *x: None)(m, r)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:161: in _post_init_wrapper
    self.post_init()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <workspace.PaddleHubEncoder object at 0x7f0a59997b10>

    def post_init(self):
        super().post_init()
        import paddlehub as hub
        module = hub.Module(name=self.model_name)
>       inputs, outputs, self.model = module.context(trainable=False)
E       AttributeError: 'MockModule' object has no attribute 'context'

__init__.py:70: AttributeError
----------------------------- Captured stdout call -----------------------------
PaddleHubEncode@12[I]:post initiating, this may take some time...
PaddleHubEncode@12[I]:post initiating, this may take some time takes 0.001 secs
___________ test_imagepaddlehubencoder_encode_with_pretrained_model ____________

    def test_imagepaddlehubencoder_encode_with_pretrained_model():
>       _test_imagepaddlehubencoder_encode()

tests/test_paddlehub.py:83: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_paddlehub.py:42: in _test_imagepaddlehubencoder_encode
    encoder = get_encoder()
tests/test_paddlehub.py:31: in get_encoder
    return PaddleHubEncoder(metas=metas)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:58: in __call__
    getattr(obj, '_post_init_wrapper', lambda *x: None)(m, r)
/usr/local/lib/python3.7/site-packages/jina/executors/__init__.py:161: in _post_init_wrapper
    self.post_init()
__init__.py:69: in post_init
    module = hub.Module(name=self.model_name)
/usr/local/lib/python3.7/site-packages/paddlehub/module/module.py:102: in __new__
    name=name, version=version, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'paddlehub.module.module.Module'>, name = 'xception71_imagenet'
version = None, kwargs = {}
fp_lock = <_io.TextIOWrapper name='/root/.paddlehub/cache/xception71_imagenet' mode='a' encoding='UTF-8'>
log_msg = 'Installing xception71_imagenet module'
extra = {'command': 'install'}, result = False
tips = 'Request Hub-Server unsuccessfully, please check your network.'
module_dir = None

    @classmethod
    def init_with_name(cls, name, version=None, **kwargs):
        fp_lock = open(os.path.join(CACHE_HOME, name), "a")
        lock.flock(fp_lock, lock.LOCK_EX)
        log_msg = "Installing %s module" % name
        if version:
            log_msg += "-%s" % version
        logger.info(log_msg)
        extra = {"command": "install"}
        result, tips, module_dir = default_module_manager.install_module(
            module_name=name, module_version=version, extra=extra)
        if not result:
            logger.error(tips)
>           raise RuntimeError(tips)
E           RuntimeError: Request Hub-Server unsuccessfully, please check your network.

/usr/local/lib/python3.7/site-packages/paddlehub/module/module.py:174: RuntimeError
----------------------------- Captured stdout call -----------------------------
PaddleHubEncode@12[I]:post initiating, this may take some time...
PaddleHubEncode@12[I]:post initiating, this may take some time takes 38.023 secs
----------------------------- Captured stderr call -----------------------------
[2020-08-25 12:46:03,293] [    INFO] - Installing xception71_imagenet module
[2020-08-25 12:46:41,315] [   ERROR] - Request Hub-Server unsuccessfully, please check your network.
=============================== warnings summary ===============================
/usr/lib/python3/dist-packages/google/protobuf/internal/api_implementation.py:154
  /usr/lib/python3/dist-packages/google/protobuf/internal/api_implementation.py:154: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from google.protobuf.pyext import _message

/usr/lib/python3/dist-packages/google/protobuf/internal/containers.py:182
  /usr/lib/python3/dist-packages/google/protobuf/internal/containers.py:182: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    MutableMapping = collections.MutableMapping

/usr/local/lib/python3.7/site-packages/jina/counter.py:6
  /usr/local/lib/python3.7/site-packages/jina/counter.py:6: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Iterator

tests/test_paddlehub.py::test_imagepaddlehubencoder_encode
tests/test_paddlehub.py::test_imagepaddlehubencoder_encode
tests/test_paddlehub.py::test_imagepaddlehubencoder_encode
  /usr/local/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)

tests/test_paddlehub.py::test_imagepaddlehubencoder_encode
  /usr/local/lib/python3.7/site-packages/paddle/fluid/framework.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Iterable

tests/test_paddlehub.py::test_imagepaddlehubencoder_encode
  /usr/local/lib/python3.7/site-packages/paddle/fluid/dygraph/dygraph_to_static/utils.py:22: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
FAILED tests/test_paddlehub.py::test_imagepaddlehubencoder_encode - Attribute...
FAILED tests/test_paddlehub.py::test_imagepaddlehubencoder_save_and_load - At...
FAILED tests/test_paddlehub.py::test_imagepaddlehubencoder_save_and_load_config
FAILED tests/test_paddlehub.py::test_imagepaddlehubencoder_encode_with_pretrained_model
======================== 4 failed, 8 warnings in 40.48s ========================
The command '/bin/sh -c pip install pytest pytest-mock mock && pytest' returned a non-zero code: 1

```